### PR TITLE
Adds in the Son of Mars ERT, replacing CLF from active rotation.

### PR DESCRIPTION
**Pull request by DominikPanic**
:cl: LaKiller8 & ArachnidNexus & MJP
imageadd: Added in new sprites and equipment for the Sons of Mars.
tweak: CLF has been replaced by the Sons of Mars in active distress rotation.
/:cl:

### CODE REVIEW

File: code/game/jobs/job/sonsofmars.dm [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+	H.equip_to_slot_or_del(new /obj/item/stack/sheet/metal/small_stack, SLOT_IN_BACKPACK)
+	H.equip_to_slot_or_del(new /obj/item/storage/box/MRE, SLOT_IN_BACKPACK)
+	H.equip_to_slot_or_del(new /obj/item/storage/box/MRE, SLOT_IN_BACKPACK)
+	H.equip_to_slot_or_del(new /obj/item/radio, SLOT_IN_BACKPACK)
+	H.equip_to_slot_or_del(new /obj/item/tool/crowbar/red, SLOT_IN_BACKPACK)
+	H.equip_to_slot_or_del(new /obj/item/storage/box/m94, SLOT_IN_BACKPACK)
+	H.equip_to_slot_or_del(new /obj/item/storage/box/m94, SLOT_IN_BACKPACK)
+	H.equip_to_slot_or_del(new /obj/item/explosive/grenade/incendiary/molotov, SLOT_IN_BACKPACK)
+	H.equip_to_slot_or_del(new /obj/item/explosive/grenade/incendiary/molotov, SLOT_IN_BACKPACK)
+	H.equip_to_slot_or_del(new /obj/item/weapon/gun/pistol/highpower, SLOT_IN_BACKPACK)
```
Comment (Ghommie) [inline code/game/jobs/job/sonsofmars.dm]: Is it github fooling me or are you giving them 3 highpower pistols like if they were some sort of bucaneers?
Also they don't seem to have any spare m16 mags.

File: code/game/jobs/job/sonsofmars.dm [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+	H.equip_to_slot_or_del(new /obj/item/weapon/gun/pistol/highpower, SLOT_IN_BACKPACK)
+	H.equip_to_slot_or_del(new /obj/item/weapon/gun/pistol/highpower, SLOT_IN_BACKPACK)
```
Comment (Ghommie) [inline code/game/jobs/job/sonsofmars.dm]: Same here.

File: code/game/jobs/job/sonsofmars.dm (hunk 1 above)
Comment (DominikPanic) [reply code/game/jobs/job/sonsofmars.dm]: It's copypasta'd CLF which would make it my fault from about 4 months ago

File: code/game/objects/items/radio/encryptionkey.dm [hunk 3]
```diff
@@ -145,3 +145,8 @@
 /obj/item/encryptionkey/imperial
 	name = "\improper Imperial encryption key"
 	channels = list("Imperial" = TRUE)
+
+
+/obj/item/encryptionkey/som
+	name = "lore som encryption key"
```
[the remark this answers is not in the document]
Comment (DominikPanic) [reply code/game/objects/items/radio/encryptionkey.dm]: True I'm just waiting for JPR for some l o r e

File: code/game/jobs/job/sonsofmars.dm (hunk 1 above)
Comment (Ghommie) [reply code/game/jobs/job/sonsofmars.dm]: Well, they'll need some different guns, otherwise they may end up looking like CLF, yet better armored.

**Comment by Hughgent** [comment]
If you would like. check out `modules/codex/codex_entries/misc_codex.dm` and the zippy pizza entry for adding the lore to the codex entries directly. lore_text is green and mechanics_text is blueish.

File: code/datums/emergency_calls/sonsofmars.dm [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+/datum/emergency_call/som/print_backstory(mob/living/carbon/human/H)
+	to_chat(H, "<b>Lore.</b>")
+
+
+/datum/emergency_call/som/create_member(datum/mind/M)
+	var/turf/spawn_loc = get_spawn_point()
+	var/mob/original = M.current
+
+	if(!istype(spawn_loc))
```
Comment (Rohesie) [inline code/datums/emergency_calls/sonsofmars.dm]:
```suggestion
	if(!spawn_loc)
```

**Comment by Pariah919** [comment]
Neat.

File: code/datums/emergency_calls/sonsofmars.dm [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+	if(original)
+		qdel(original)
+
+	print_backstory(H)
+
+	if(!leader)
+		leader = H
+		var/datum/job/J = SSjob.GetJobType(/datum/job/som/leader)
+		J.equip(H)
+		to_chat(H, "<span class='notice'>Lore.</span>")
```
Comment (Rohesie) [inline code/datums/emergency_calls/sonsofmars.dm]: Awaiting lore.

**Comment by Rohesie** [comment]
Merge conflict, otherwise ready to go in.

**Pull request merged**